sideboard/websockets.py

```diff
@@ -522,6 +569,7 @@ def send(self, **message):
         """
         if self.is_closed:
             log.debug('ignoring send on an already closed websocket: {}', message)
+            self.unsubscribe_all()
```

Comment:
See above; this is the same performance improvement I describe in my previous comment.